content/en/docs/contribute/analytics.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Viewing site analytics
+content_type: concept
+weight: 100
+card:
+  name: contribute
+  weight: 100
+---
+
+<!-- overview -->
+
+This page contains information about the kubernetes.io analytics dashboard.
+
+
+<!-- body -->
+
+[View the dashboard.](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC/edit)
```

Comment:
nit: i believe it shouldn't have the edit at the end, since it's user facing.

Reply:
```suggestion
View the [dashboard](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC/edit).
```

Reply:
updated the link here.